keps/sig-network/563-dual-stack/README.md

```diff
@@ -1362,23 +1356,25 @@ This capability will move to stable when the following criteria have been met.
   the enablement)?**
   Yes.
 
+  [TODO] If we disable the featuregate, do I also have to remove the CLI parameters or are those still accepted?
+
```

Comment:
This came from feedback and I think we could clarify it here.

Reply:
The full question:
_If I disable the featuregate, do I also have to remove the CLI parameters or are those still accepted? When I re-enable the featuregate, there may be existing services configured as dual-stack. Are the cluster IPs allocated at that time? How do I coordinate the enablement and re-enablement? That is, presumably you roll out the enablement on kube-apiserver first, then other components later. During the transition, secondary family addresses will show in the API server but won't work yet until the pods are re-created with dual family after kubelet is re-enabled, right? So you will have a service that may return an IPv6 address (say) from DNS for the cluster IP but there are no endpoints for it yet? Are we doing anything to mitigate that (I can't recall if we return the cluster IP from DNS if there are no endpoints, maybe we don't)? Or can we at least document this so that the operator knows that the gap here? It's not likely to be a huge issue for upgrade/initial enablement because there won't be dual stack services yet, but re-enablement I can see some disruption._

Reply:
Yes, if disabling the gate we have to remove CLI parameters.

Reply:
An older client won't see the new fields.